Makefile

```diff
@@ -636,6 +666,11 @@ $(BUILD_DIR)/src/libultra/libc/ll.o: OPTFLAGS := -Ofast
 $(BUILD_DIR)/src/overlays/%.o: CFLAGS += -fno-merge-constants -mno-explicit-relocs -mno-split-addresses
 endif
 
+SET_ABI_BIT = @:
```

Comment:
Since the implementation is changing, just thought I would mention. We may be able to remove the set_o32abi_bit script and just use `--no-warn-mismatch` with the linker instead.
Tbh not sure if this would be better or not, but just wanted to throw it out as an option.

Reply:
In the future I'd actually like to be even stricter with linker checking, as in I'd like to eventually remove `--no-check-sections --accept-unknown-input-arch`. So I think this option is more in-line with that longer-term goal